patches/api/0352-steer-vehicle-event.patch

```diff
@@ -0,0 +1,78 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: rgnter <maros.prejsa1@gmail.com>
+Date: Mon, 3 Jan 2022 17:56:28 +0100
+Subject: [PATCH] steer vehicle event
+
+
+diff --git a/src/main/java/com/destroystokyo/paper/event/player/PlayerSteerVehicleEvent.java b/src/main/java/com/destroystokyo/paper/event/player/PlayerSteerVehicleEvent.java
+new file mode 100644
+index 0000000000000000000000000000000000000000..dbe465517e3c75d2baa99a34db30d7b4d3604534
+--- /dev/null
++++ b/src/main/java/com/destroystokyo/paper/event/player/PlayerSteerVehicleEvent.java
+@@ -0,0 +1,66 @@
++package com.destroystokyo.paper.event.player;
++
++import org.bukkit.event.Event;
++import org.bukkit.event.HandlerList;
++import org.jetbrains.annotations.NotNull;
++
++/**
++ * Called when player is steering any vehicle.
++ */
++public class PlayerSteerVehicleEvent extends Event {
```

Comment:
This should extend PlayerEvent, we have no way to actually get the player who steered this.